tests/unit/alp_gemm.cpp

```diff
@@ -35,26 +56,107 @@ void alp_program( const size_t & unit, alp::RC & rc ) {
 	size_t N = 20 * unit;
 	size_t K = 30 * unit;
 
-	// dimensions of views over A, B and C
-	size_t m = unit;
-	size_t n = 2 * unit;
-	size_t k = 3 * unit;
-
 	alp::Matrix< double, structures::General > A( M, K );
 	alp::Matrix< double, structures::General > B( K, N );
 	alp::Matrix< double, structures::General > C( M, N );
 
-	Scalar< double > alpha( 0.5 );
-	Scalar< double > beta( 1.5 );
-
-	rc = algorithms::gemm_like_example(
-		m, n, k,
-		alpha,
-		A, 1, 1, 1, 1,
-		B, 2, 1, 2, 4,
-		beta,
-		C, 0, 0, 1, 1,
-		ring );
+	// Initialize containers A, B, C, alpha, beta
+	constexpr double A_value = 2;
+	constexpr double B_value = 3;
+	constexpr double C_value = 4;
+	rc = rc ? rc : alp::set( A, alp::Scalar< double >( A_value ) );
```

Comment:
Would this (and next line) make it harder for us to assess the correct functioning of transposition?

Reply:
Implemented matrix initialization with random values.